### PR TITLE
[BI-645] - Make trait import case insensitive

### PR DESCRIPTION
**Pull request by ctucker3**

### CODE REVIEW

File: src/test/java/org/breedinginsight/api/v1/controller/TraitUploadControllerIntegrationTest.java [hunk 1]
```diff
@@ -610,6 +610,33 @@ void postTraitUploadDuplicateTraits() {
...
+        JsonObject result = JsonParser.parseString(uploadResponse.body()).getAsJsonObject().getAsJsonObject("result");
+
+        checkValidTraitUpload(result);
+    }
+
+    @Test
+    @Order(10)
+    void putTraitUploadDuplicateMismatchedHeaders() {
+
+        // Traits should be inserted just fine
```
Comment (nickpalladino) [inline src/test/java/org/breedinginsight/api/v1/controller/TraitUploadControllerIntegrationTest.java:631]: doesn't seem accurate

File: src/test/resources/files/data_duplicate_headers.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+trait name,TRAiT NnME,Trait Abbreviations,trAit synoNyms,trait Description,Trait level,Trait status,Trait lists,Method name,Method description,Method class,Method formula,Scale name,Scale class,Scale decimal places,Scale lower limit,ScALe Upper limit,Scale CAtegories
```
Comment (nickpalladino) [inline src/test/resources/files/data_duplicate_headers.csv:1]: Is trait name supposed to be duplicate?

Comment (ctucker3) [reply src/test/resources/files/data_duplicate_headers.csv]: Yep, good catch. Changed and pushed

File: src/test/java/org/breedinginsight/api/v1/controller/TraitUploadControllerIntegrationTest.java (hunk 1 above)
Comment (ctucker3) [reply src/test/java/org/breedinginsight/api/v1/controller/TraitUploadControllerIntegrationTest.java]: Haha nope! Removed it

**Review by nickpalladino — APPROVED** [review]

**Review by DebWeigand — APPROVED** [review]

**Pull request merged**